From 377da6b9998e3fd6395b854999ab2531c54820ed Mon Sep 17 00:00:00 2001
From: BigWhiteDog <legend.z@foxmail.com>
Date: Tue, 29 Mar 2022 15:25:06 +0800
Subject: [PATCH] a cross compile patch for riscv64

---
 Makefile   | 2 +-
 redis.conf | 6 +++---
 2 files changed, 4 insertions(+), 4 deletions(-)

diff --git a/Makefile b/Makefile
index e614ede..68f95dc 100644
--- a/Makefile
+++ b/Makefile
@@ -3,7 +3,7 @@
 default: all
 
 .DEFAULT:
-	cd src && $(MAKE) $@
+	cd src && $(MAKE) CC=riscv64-linux-gnu-gcc MALLOC=libc $@
 
 install:
 	cd src && $(MAKE) $@
diff --git a/redis.conf b/redis.conf
index e8eff27..f4d2c94 100644
--- a/redis.conf
+++ b/redis.conf
@@ -72,7 +72,7 @@
 # IF YOU ARE SURE YOU WANT YOUR INSTANCE TO LISTEN TO ALL THE INTERFACES
 # JUST COMMENT OUT THE FOLLOWING LINE.
 # ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
-bind 127.0.0.1 -::1
+#bind 127.0.0.1 -::1
 
 # Protected mode is a layer of security protection, in order to avoid that
 # Redis instances left open on the internet are accessed and exploited.
@@ -91,7 +91,7 @@ bind 127.0.0.1 -::1
 # you are sure you want clients from other hosts to connect to Redis
 # even if no authentication is configured, nor a specific set of interfaces
 # are explicitly listed using the "bind" directive.
-protected-mode yes
+protected-mode no
 
 # Accept connections on the specified port, default is 6379 (IANA #815344).
 # If port 0 is specified Redis will not listen on a TCP socket.
@@ -254,7 +254,7 @@ tcp-keepalive 300
 # By default Redis does not run as a daemon. Use 'yes' if you need it.
 # Note that Redis will write a pid file in /var/run/redis.pid when daemonized.
 # When Redis is supervised by upstart or systemd, this parameter has no impact.
-daemonize no
+daemonize yes
 
 # If you run Redis from upstart or systemd, Redis can interact with your
 # supervision tree. Options:
-- 
2.35.1

