From c53706efe3c60d22bfd5211ec17e7edbb28753c6 Mon Sep 17 00:00:00 2001
From: BigWhiteDog <legend.z@foxmail.com>
Date: Tue, 29 Mar 2022 15:25:06 +0800
Subject: [PATCH] a cross compile patch for riscv64

---
 Makefile | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/Makefile b/Makefile
index e614ede..68f95dc 100644
--- a/Makefile
+++ b/Makefile
@@ -3,7 +3,7 @@
 default: all
 
 .DEFAULT:
-	cd src && $(MAKE) $@
+	cd src && $(MAKE) CC=riscv64-linux-gnu-gcc MALLOC=libc $@
 
 install:
 	cd src && $(MAKE) $@
-- 
2.35.1

